web/app/DeviceImages.js

```diff
@@ -67,6 +70,14 @@ Ext.define('Traccar.DeviceImages', {
                 encodeURIComponent(new XMLSerializer().serializeToString(svg.documentElement));
     },
 
+    cloneDocument: function (document) {
```

Comment:
I would rename "document" to something else because "document" is a global object in java.